README.md

```diff
@@ -59,75 +59,146 @@ It is possible to change the behaviour of the data builder by defining environme
 - (Optional) "NOCLEANUP" can be used to disable removal of historical data in storage
 - (Optional) "JAVA_OPTS" Java parameters for running OTP
 
-#### Data processing steps
+### Data processing steps
 
-Seed data can be retrieved with a single gulp command:
-
-1. `seed`
+- `seed`
 
 Downloads previous data from storage (env variable SEED_TAG can be used to customize which storage location is used)
-and extracts osm, dem and gtfs data from there and places it in 'data/seed' and 'data/ready' directories.
-Old data acts as backup in case fetching/validating new data fails.
+and then extracts osm, dem, and gtfs data and places it in the `data/seed` and `data/ready` directories.
+The old data acts as backup in case fetching/validating new data fails. The command uses the zipped contents of the latest build that built a complete graph (from prebuilt data or with no split build).
+
+- `dem:update`
+
+This command downloads required DEM information, after which data is copied to the `data/downloads/dem` directory.
```

Comment:
Btw I noticed it's easy to misinterpret docs from this section as if you start reading from the middle of this, it's not clear if the description is for the command above or below. I'm not quite sure what would be the best way to avoid this confusion.

Reply:
I made this more clear